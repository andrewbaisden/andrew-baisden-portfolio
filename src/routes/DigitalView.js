import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPageDigitalView from '../components/PortfolioPageDigitalView';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
// import PortfolioCarouselDigitalView from '../components/PortfolioCarouselDigitalView';

const DigitalView = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="I joined Digital View as Junior Web Developer through an Internship work programme at The University of West London. I was part of a team of 4 including myself."
        />
        <title>Digital View - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/portfolio/digital-view" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPageDigitalView companyBG="#FFFFFF" />
    {/* <PortfolioCarouselDigitalView /> */}
    <Footer />
  </div>
);

export default DigitalView;
