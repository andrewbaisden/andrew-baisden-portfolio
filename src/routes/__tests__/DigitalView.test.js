import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import DigitalView from '../DigitalView';

Enzyme.configure({ adapter: new Adapter() });

describe('<DigitalView />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<DigitalView />);
    expect(wrapper).toMatchSnapshot();
  });
});
