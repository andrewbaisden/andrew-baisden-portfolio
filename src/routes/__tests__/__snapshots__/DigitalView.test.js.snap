// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DigitalView /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
